src/SettingsWidget.py

```diff
@@ -132,7 +140,21 @@ def _update_leagues_combo(self) -> None:
             index += 1
         self.combo_league.setCurrentText(self.league)
 
-    def load_main_widget_y(self) -> int:
+    def _update_mod_file_combo(self) -> None:
```

Comment:
I'm not sure if you can reproduce it on Linux, but on Windows there is small issue with updating this combo. If you don't move settings widget at all, everything works fine. But if you moved it before changing mod file in combo, on each combo update whole settings widget changes its location a bit.

Reply:
Bug is coming from parent class DragWidget, nothing wrong here.